keps/sig-node/3063-dynamic-resource-allocation/kep.yaml

```diff
@@ -24,11 +24,11 @@ stage: alpha
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.26"
+latest-milestone: "v1.27"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
-  alpha: "v1.26"
+  alpha: "v1.27"
```

Comment:
Shouldn't this have been left at the (now historic) milestone of reaching alpha in 1.26?